CONTRIBUTING.md

```diff
@@ -56,11 +56,12 @@ pnpm start
 1. Read the changelogs and release notes of the dependencies you'd like to
    upgrade. Look for potential breaking changes, and for bug fixes and new
    features that may help improve the codebase.
-1. Run `pnpm up -L <pkg-name> --filter <project-name>` to upgrade a dependency
-   to the latest version in a given project. Alternatively, you can also edit
-   the relevant `package.json` file(s) manually and run `pnpm install`, but make
-   sure to specify an exact dependency version rather than a range (i.e. don't
-   prefix the version with a caret or a tilde).
+1. Run `pnpm up -r -L <pkg-name>` to update a dependency to the latest version
+   in all projects. Alternatively, you can either replace `-r` with `--filter`
+   to target specific projects, or edit the relevant `package.json` file(s)
+   manually and run `pnpm install` (but make sure to specify an exact dependency
+   version rather than a range - i.e. don't prefix the version with a caret or a
+   tilde).
```

Comment:
Switching the `--filter` flag with the `-r` flag, as the most common scenario by far is to update a dependency to the latest version in every project that has it.